src/components/ward-management/ward-management-modes.module.css

```
@@ -32,13 +32,15 @@
   --ward-leading-body: 1.4;
   --ward-leading-relaxed: 1.45;
   --ward-leading-prose: 1.5;
+  --ward-stroke-thin: 0.0625rem;
+  --ward-stroke-accent: 0.1875rem;
   --ward-radius-round: 50%;
   --ward-z-base: 1;
   --ward-z-phone: 20;
   display: grid;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)